CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
-project (ignition-tools2)
+project (gz-tools2)
 
-# Note about ignition-cmake:
+# Note about gz-cmake:
 # -------------------------
-# The project is not using ignition-cmake2 due to problems
-# with compatibility when mixed with projects using ignition-cmake0
-# When ignition-cmake0 will be EOL the code to implement
-# ignition-cmake2 back to this project can be found at:
+# The project is not using gz-cmake2 due to problems
+# with compatibility when mixed with projects using gz-cmake0
+# When gz-cmake0 will be EOL the code to implement
+# gz-cmake2 back to this project can be found at:
 # https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-tools/pull-requests/6
 
 string (SUBSTRING ${PROJECT_NAME} 9 -1 IGN_PROJECT_NAME)
```

Comment:
Be wary of this! I think the indexing will change.
Also it would be good to edit IGN_PROJECT_NAME, it's an output variable as per [the docs](https://cmake.org/cmake/help/latest/command/string.html#substring)

Reply:
I'm wondering if a regex find would be more portable...

Reply:
This goes away with #96, how about we get that one in first? I believe it doesn't break anything downstream

Reply:
:+1:!